src/main/java/net/coderbot/iris/gl/program/ProgramBuilder.java

```diff
@@ -5,12 +5,28 @@
 import net.coderbot.iris.gl.shader.ProgramCreator;
 import net.coderbot.iris.gl.shader.ShaderConstants;
 import net.coderbot.iris.gl.shader.ShaderType;
+import net.coderbot.iris.gl.shader.StandardMacros;
 import org.jetbrains.annotations.Nullable;
+import org.lwjgl.opengl.GL20C;
 import org.lwjgl.opengl.GL21C;
 
+import static net.coderbot.iris.gl.shader.StandardMacros.*;
```

Comment:
`import static` should generally be avoided since it makes it unclear where things are being called from